Cargo.toml

```diff
@@ -22,8 +22,8 @@ spectre = { git = "https://github.com/niklaslong/spectre", rev = "f1d6698" }
 tabled = "0.10"
 time = "0.3"
 toml = "0.6.0"
-ziggurat-core-metrics = { git = "https://github.com/runziggurat/ziggurat-core", tag = "v0.1.2-zgm" }
-ziggurat-core-crawler = { git = "https://github.com/runziggurat/ziggurat-core", rev = "001c943" }
+ziggurat-core-metrics = { git = "https://github.com/kylegranger/ziggurat-core", rev = "d8e4e0a" }
+ziggurat-core-crawler = { git = "https://github.com/kylegranger/ziggurat-core", rev = "d8e4e0a" }
```

Comment:
Shouldn't point to the main ziggurat-core repository instead of user one?

Reply:
Fixed!  I had the correct revision, but older path slipped in.